hera_cal/tests/test_vis_clean.py

```diff
@@ -103,6 +115,80 @@ def test_read_write(self):
         assert os.path.exists("ex.uvh5")
         os.remove('ex.uvh5')
 
+    @pytest.mark.filterwarnings("ignore:.*dspec.vis_filter will soon be deprecated")
```

Comment:
We probably don't want to filter deprecation warnings... just makes it easier to accumulate technical debt.

Reply:
I've removed this.